### ISSUE TITLE
Add French for marginal tidiers

### ISSUE THREAD

**Issue by ddsjoberg** [ddsjoberg/gtsummary#1417]
@larmarange i was reviewing your guidebook for R (as much as I could as a non-French speaker), and it is amazing!

I noticed that the headers for the new broom.helpers margin tidiers do not have translations. If you add the translations here in this issue, I'll incorporate them into the gtsummary

**Comment by larmarange** [comment]
Yes I will.  I'm still thinking about changing the name for tidy_all_effects(). Marginal means may be more appropriate.

I should add tidiers for the marginaleffect package as well and try to have a more consistent naming

**Comment by larmarange** [comment]
Here is the full list:

- "Average Marginal Effects" = "Effets Marginaux Moyens"
- "Marginal Effects at the Mean" = "Effets Marginaux à la Moyenne"
- "Marginal Effects at Marginal Means" = "Effets Marginaux aux Moyennes Marginales"
- "Marginal Effects" = "Effets Marginaux"
- "Average Marginal Contrasts" = "Contrastes Marginaux Moyens"
- "Marginal Contrasts at the Mean" = "Contrastes Marginaux à la Moyenne"
- "Marginal Contrasts at Marginal Means" = "Contrastes Marginaux aux Moyennes Marginales"
- "Marginal Contrasts" = "Contrastes Marginaux"
- "Marginal Means" = "Moyennes Marginales"
- "Average Marginal Predictions" = "Prédictions Marginales Moyennes"
- "Marginal Predictions at the Mean" = "Prédictions Marginales à la Moyenne"
- "Marginal Predictions at Marginal Means" = "Prédictions Marginales aux Moyennes Marginales"
- "Marginal Predictions" = "Prédictions Marginales"

**Comment by ddsjoberg** [comment]
Hey hey @larmarange ! I see you're prepping for a broom.helpers release. I think gtsummary could benefit from a release afterwards.

The text above will appear in the `tbl_regression()` headers, right? (Just want to make sure I add the translations to the correct place).

**Comment by larmarange** [comment]
Like **OR** for logistic regression coefficients

**Comment by ddsjoberg** [comment]
Thanks @larmarange ! Can you take a look at this PR https://github.com/ddsjoberg/gtsummary/pull/1479 ?
